src/types.jl

```diff
@@ -37,7 +37,7 @@ SCSMatrix(m::ManagedSCSMatrix) =
     SCSMatrix(pointer(m.values), pointer(m.rowval), pointer(m.colptr), m.m, m.n)
 
 
-struct SCSSettings
+mutable struct SCSSettings
```

Comment:
I don't think this change is valid. `SCSSettings` needs to mirror the C struct in order for the `ccall`s to be correct. `mutable struct`s can't be used to mirror C structs AFAIK.